gcovr/coverage.py

```diff
@@ -48,16 +48,19 @@
 
 def sort_coverage(
     covdata: CovData,
-    by_branch: bool,
-    by_num_uncovered: bool,
-    by_percent_uncovered: bool,
     reverse: bool,
+    *, # end of positional arguments
+    by_branch: bool = False,
+    by_decision: bool = False,
+    by_num_uncovered: bool = False,
+    by_percent_uncovered: bool = False,
```

Comment:
The forced keyword arguments is a good idea but the intention here is that all arguments need to be supplied and this would not be possible anymore if we use default values.
```suggestion
    by_branch: bool,
    by_decision: bool,
    by_num_uncovered: bool,
    by_percent_uncovered: bool,
```

Reply:
This would mean that a lot of other writers will need a change. But still if this is desired, I can do this.